.config/CredScanSuppressions.json

```diff
@@ -0,0 +1,13 @@
+{
+    "tool": "Credential Scanner",
+    "suppressions": [
+        {
+            "_justification": "Unit tests use dummy input data.",
+            "file": [
+                "test/NuGet.Core.Tests/NuGet.Protocol.Tests/Plugins/Messages/GetCredentialsResponseTests.cs",
```

Comment:
It looks there 2 ways of suppression, [inline or suppression file](https://strikecommunity.azurewebsites.net/articles/8148/handling-false-positives-and-suppressions.html).

Reply:
Without suppression file it'll [fail](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5284233&view=logs&j=d80b2849-30d4-52dd-74cd-d6536ba98fe0&t=1827c9b6-09e3-515b-9a63-74df1158657e&l=15).

Reply:
Below are actual offending lines:

https://github.com/NuGet/NuGet.Client/edit/dev/docs/cross-platform-debugging.md#L66

https://github.com/NuGet/NuGet.Client/blob/966b1b21eec28834365a8604744dab682c8326ea/test/NuGet.Core.Tests/NuGet.Protocol.Tests/Plugins/Messages/GetCredentialsResponseTests.cs#L72-L73
https://github.com/NuGet/NuGet.Client/blob/966b1b21eec28834365a8604744dab682c8326ea/test/NuGet.Core.Tests/NuGet.Protocol.Tests/Plugins/Messages/GetCredentialsResponseTests.cs#L90-L91
https://github.com/NuGet/NuGet.Client/blob/966b1b21eec28834365a8604744dab682c8326ea/test/NuGet.Core.Tests/NuGet.Protocol.Tests/Plugins/Messages/GetCredentialsResponseTests.cs#L109-L112

https://github.com/NuGet/NuGet.Client/blob/966b1b21eec28834365a8604744dab682c8326ea/test/NuGet.Core.Tests/NuGet.Protocol.Tests/Plugins/Messages/SetCredentialsRequestTests.cs#L30 (CredScanner detect as line 37)
https://github.com/NuGet/NuGet.Client/blob/966b1b21eec28834365a8604744dab682c8326ea/test/NuGet.Core.Tests/NuGet.Protocol.Tests/Plugins/Messages/SetCredentialsRequestTests.cs#L49
https://github.com/NuGet/NuGet.Client/blob/966b1b21eec28834365a8604744dab682c8326ea/test/NuGet.Core.Tests/NuGet.Protocol.Tests/Plugins/Messages/SetCredentialsRequestTests.cs#L72